### PR TITLE
Fix syntax warning over comparison of literals using is.

### LINKED ISSUE #176: SyntaxWarnings with Python 3.8
Hi,

I've recently updated to Python 3.8 and python-memcached now emits 2 syntax warnings:

```
memcache.py:1303: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if key is '':
memcache.py:1304: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if key_extra_len is 0:
```

Would be great if you could fix this… and publish a patch-version on pypi!

Best regards

### PR DESCRIPTION
**Pull request by tirkarthi**
Fixes #176

### CODE REVIEW

**Review by ericxiett — COMMENTED** [review]
LGTM

**Comment by victorelec14** [comment]
The same problem. Please merge this PR :)

**Review by DAcodedBEAT — APPROVED** [review]

**Review by victorelec14 — APPROVED** [review]

File: memcache.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ class _ConnectionDeadError(Exception):
 
 
 class Client(threading.local):
-    """Object representing a pool of memcache servers.
+    r"""Object representing a pool of memcache servers.
```
[the remark this answers is not in the document]
Comment (DAcodedBEAT) [reply memcache.py:110]: I believe this is valid.

>Both string and bytes literals may optionally be prefixed with a letter 'r' or 'R'; such strings are called raw strings and treat backslashes as literal characters. As a result, in string literals, '\U' and '\u' escapes in raw strings are not treated specially. Given that Python 2.x’s raw unicode literals behave differently than Python 3.x’s the 'ur' syntax is not supported.

source: https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

**Review by arghodayah — APPROVED** [review]

File: memcache.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (linsomniac) [reply memcache.py:110]: I'm going to remove that \ in the docstring which should resolve this without the 'r'.  I don't believe the \ is needed, it probably was an artifact of a continuation line.  Your is -> == is included in a patch I just accepted, so I'm going to commit this change and close this PR as fixed, thank you for the PR!

**Comment by linsomniac** [comment]
is -> == done by another PR, removing spurious \ in docstring rather than converting to r string.

**Pull request closed without merging**